config.h.in

```diff
@@ -189,9 +189,6 @@
 /* Define if the listen(2) system call accepts backlog value of -1. */
 #undef LISTEN_NEGATIVE_BACKLOG
 
-/* Define as the return type of signal handlers (int or void).  */
-#undef RETSIGTYPE
```

Comment:
I wonder if this should be left in, for compatibility of compilation with any third-party modules that use it?  Perhaps in the `include/compat.h` file.